apps/api/Dockerfile

```diff
@@ -15,7 +15,9 @@ EXPOSE 8000
 # --------------------------------------------------------------------------
 
 # COMMAND TO RUN
-RUN   apt-get update \
+RUN  --mount=type=secret,id=github_token \
```

Comment:
"For the duration of this run step only, mount the secret named `github_token` (which we defined in the arguments to the `docker build` command) as a file whose path will default to `/run/secrets/github_token`"